db_test.go

```diff
@@ -425,6 +425,40 @@ func (s *SurrealDBTestSuite) TestSmartMarshalQuery() {
 	})
 }
 
+// BenchmarkCreate benchmarks the creation of a record
+func BenchmarkCreate(b *testing.B) {
```

Comment:
We could probably have benchmarks in another file or package as they won't be run as often or under the same circumstances as unit and integration tests